.github/workflows/release_mondoo_pkgs.yaml

```diff
@@ -29,11 +29,12 @@ jobs:
       - name: Verify valid version
         id: version
         run: |
-          if [[ ! $VERSION =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+          if [[ ! $VERSION =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Comment:
I need to revert that because we need the correct version here. I first thought we can do it here but package versions cannot have a prefix so we need to remove it before otherwise the package build generate invalid versions

Reply:
Ah I take it all back